waspc/e2e-test/test-outputs/waspComplexTest-golden/waspComplexTest/.wasp/out/server/src/auth/providers/config/google.ts

```diff
@@ -11,7 +11,7 @@ const _waspUserDefinedConfigFn = undefined
 const _waspOAuthConfig: OAuthConfig = {
     clientID: process.env.GOOGLE_CLIENT_ID,
     clientSecret: process.env.GOOGLE_CLIENT_SECRET,
-    scope: ['"profile"'],
+    scope: ['profile'],
```

Comment:
I ran the e2e tests with `accept all` blindly and it negated the usefulness of e2e tests